crates/tools/js/benchmark/solidity-tests.js

```diff
@@ -56,19 +56,25 @@ async function runForgeStdTests(forgeStdRepoPath) {
     path.join(forgeStdRepoPath, "hardhat.config.js"),
   );
 
-  const testSuites = listFilesRecursively(artifactsDir)
-    .filter((p) => !p.endsWith(".dbg.json") && p.includes(".t.sol"))
-    .map(loadContract.bind(null, hardhatConfig))
-    .filter((ts) => !EXCLUDED_TEST_SUITES.has(ts.id.name));
+  const artifacts = listFilesRecursively(artifactsDir)
+    .filter((p) => !p.endsWith(".dbg.json") && !p.includes("build-info"))
```

Comment:
There are two types of files in the artifact directory that are not contract artifacts:
- debug info files (ends with `.dbg.json`)
- build info files (is of the format `build-info/[0-9a-f]+.json`)

Previously we filtered paths with the condition `!p.endsWith(".dbg.json") && p.includes(".t.sol")`, because we only wanted test artifacts. This implicitly excluded build info files. But now we want all contract artifacts, so I removed the `p.includes(".t.sol")` condition and replaced it with `!p.includes("build-info")`.

Reply:
Yep that'd break it. But this script controls how the project is built